### PR TITLE
Fix maxshield trans

### PR DESCRIPTION
**Pull request by agramfort**
finishing #289

### CODE REVIEW

File: mne_bids/tests/test_read.py [hunk 1]
```diff
@@ -125,3 +126,32 @@ def test_handle_events_reading():
...
+    import nibabel as nib
+
+    ctf_data_path = op.join(testing.data_path(), 'CTF')
+    raw_ctf_fname = op.join(ctf_data_path, 'testdata_ctf.ds')
+    raw_ctf = mne.io.read_raw_ctf(raw_ctf_fname)
+    output_path = _TempDir()
+    write_raw_bids(raw_ctf, bids_basename, output_path,
+                   overwrite=False)
+
+    # Take a fake trans
```
Comment (jasmainak) [inline mne_bids/tests/test_read.py]: nice idea! It's where I was confused how to proceed

**Comment by jasmainak** [comment]
@agramfort ready to merge?

**Comment by agramfort** [comment]
Yes

**Comment by jasmainak** [comment]
Thanks @agramfort and @dnacombo !

**Pull request merged**